en/orm/entities.rst

```diff
@@ -461,7 +461,7 @@ converting the associated data into the correct format.
 Exposing Virtual Properties
 ---------------------------
 
-By default virtual properties are not exported when converting entities to
+By default virtual fields are not exported when converting entities to
```

Comment:
The goal is to use the same word across the doc ? Why has this one been changed then ?